### ISSUE TITLE
Fix link to example report

### ISSUE THREAD

**Issue by fkohrt** [ropensci/daiquiri#8]
The example report linked in the [documentation](https://ropensci.github.io/daiquiri/index.html) is unavailable under

```
https://ropensci.github.io/daiquiri/misc/example_data_report.html
```

Instead, it is located at

```
https://ropensci.github.io/daiquiri/articles/example_report.html
```

**Comment by phuongquan** [comment]
@fkohrt can you tell me where in the documentation the broken link is? I cannot find it.

**Comment by fkohrt** [comment]
I believe this was solved with one of 49b023a or 2c75339
